src/test/2024-05/GPU_exp.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.10;
+
+import "forge-std/Test.sol";
+import "./../interface.sol";
+
+// Profit : ~32 USD
+// TX : https://app.blocksec.com/explorer/tx/bsc/0x2c0ada695a507d7a03f4f308f545c7db4847b2b2c82de79e702d655d8c95dadb
+// GUY : https://twitter.com/PeckShieldAlert/status/1788153869987611113
+// Vuln Contract: https://bscscan.com/address/0xf51cbf9f8e089ca48e454eb79731037a405972ce
+
```

Comment:
Analysis: Due to a logic flaw in the transfer mechanism, it's possible to achieve exponential growth with only a small amount of tokens.